person.rb

```diff
@@ -1,18 +1,31 @@
 require 'securerandom'
-
-class Person
+require_relative './nameable'
+require_relative './capitalize_decorator'
+require_relative './trimmer_decorator'
+require_relative './rental'
+class Person < Nameable
   attr_accessor :name, :age
```

Comment:
- Please, let's also move the  `:rentals` to the `attr_accessor` as this is the intercessor between a `Person` and a  `Book` denoting the `many-to-many` (also `has-many-through`) relationship between `Person` and `Book`.